src/components/Player/SongScore.svelte

```diff
@@ -23,299 +16,73 @@
-                        <span slot="label">
-                          <Value
-                            title={`Left accuracy: ${comparePlayer.beatSavior.stats.leftAverageCut ? comparePlayer.beatSavior.stats.leftAverageCut.map(v => (configStore, $configStore, formatNumber(v))).join('/') : ''}`}
-                            value="{comparePlayer.beatSavior.stats.accLeft}"
-                            inline={false} digits={2}
-                          />
-                        </span>
-                    </Badge>
-                  </span>
-              {/if}
-
-              {#if comparePlayer.beatSavior.stats.accRight}
-                  <span class="beatSavior with-badge compare">
-                    <Badge onlyLabel={true} color="white" bgColor={ssrConfig.rightSaberColor}>
-                        <span slot="label">
-                          <Value
-                            title={`Right accuracy: ${comparePlayer.beatSavior.stats.rightAverageCut ? comparePlayer.beatSavior.stats.rightAverageCut.map(v => (configStore, $configStore, formatNumber(v))).join('/') : ''}`}
-                            value="{comparePlayer.beatSavior.stats.accRight}" inline={false} digits={2}
-                          />
-                        </span>
-                    </Badge>
-                  </span>
-              {/if}
-
-              {#if comparePlayer.beatSavior.stats.miss !== undefined}
-                  <span class="beatSavior with-badge compare">
-                    <Badge onlyLabel={true} color="white" bgColor="var(--dimmed)">
-                        <span slot="label" title={`Missed notes: ${comparePlayer.beatSavior.stats.missedNotes}, Bad cuts: ${comparePlayer.beatSavior.stats.badCuts}, Bomb hit: ${comparePlayer.beatSavior.stats.bombHit}, Wall hit: ${comparePlayer.beatSavior.stats.wallHit}`}>
-                          {#if comparePlayer.beatSavior.stats.miss || comparePlayer.beatSavior.stats.bombHit || comparePlayer.beatSavior.stats.wallHit}
-                            <i class="fas fa-times"></i>
-                            <Value
-                              title={`Missed notes: ${comparePlayer.beatSavior.stats.missedNotes}, Bad cuts: ${comparePlayer.beatSavior.stats.badCuts}, Bomb hit: ${comparePlayer.beatSavior.stats.bombHit}, Wall hit: ${comparePlayer.beatSavior.stats.wallHit}`}
-                              value="{comparePlayer.beatSavior.stats.miss}" inline={false} digits={0}
-                            />
-                          {:else if (!comparePlayer.beatSavior.stats.wallHit && !comparePlayer.beatSavior.stats.bombHit)}
-                            FC
-                          {/if}
-                        </span>
-                    </Badge>
-                  </span>
-              {/if}
-            {/if}
-          {/each}
-        {/if}
-      </section>
+      <PlayerPerformance
+          service="{service}"
```

Comment:
Variables named the same as arguments can be simply passed as {varName}. Look at SongInfo a little higher for example